db/migrate/20190916151457_rename_loa_to_ial.rb

```diff
@@ -0,0 +1,5 @@
+class RenameLoaToIal < ActiveRecord::Migration[5.1]
+  def change
+    safety_assured { rename_column :service_provider_requests, :loa, :ial }
```

Comment:
With this change the old and new servers are not compatible which will cause issues. What we will need to do is add a new column, start writing to the new column, backfill the new column with data from the old column, stop writing to the old column, and then finally drop the old column.

Reply:
Switched to adding the column first.